includes/data-stores/class-wc-admin-reports-taxes-stats-data-store.php

```diff
@@ -0,0 +1,225 @@
+<?php
+/**
+ * WC_Admin_Reports_Taxes_Stats_Data_Store class file.
+ *
+ * @package WooCommerce Admin/Classes
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+
+/**
+ * WC_Reports_Taxes_Stats_Data_Store.
+ */
+class WC_Admin_Reports_Taxes_Stats_Data_Store extends WC_Admin_Reports_Data_Store implements WC_Admin_Reports_Data_Store_Interface {
+
+	/**
+	 * Table used to get the data.
+	 *
+	 * @var string
+	 */
+	const TABLE_NAME = 'wc_order_tax_lookup';
+
+	/**
+	 * Mapping columns to data type to return correct response types.
+	 *
+	 * @var array
+	 */
+	protected $column_types = array(
+		'tax_codes'    => 'intval',
+		'total_tax'    => 'floatval',
+		'order_tax'    => 'floatval',
+		'shipping_tax' => 'floatval',
+		'order_count'  => 'intval',
+	);
+
+	/**
+	 * SQL columns to select in the db query.
+	 *
+	 * @var array
+	 */
+	protected $report_columns = array(
+		'tax_codes'    => 'COUNT(DISTINCT tax_rate_id) as tax_codes',
+		'total_tax'    => 'SUM(total_tax) AS total_tax',
```

Comment:
Nitpick (probably not specific to tax stats): should we force these total values to be to 2 decimal places?

Here's a response I got in testing:
```json
{
    "totals": {
        "tax_codes": 2,
        "total_tax": 46.370000000000005,
        "order_tax": 40.37,
        "shipping_tax": 6,
        "orders": "3"
    },
    "intervals": [
        {
            "datetime_anchor": "2018-12-07 00:00:00",
            "time_interval": "2018-49",
            "tax_codes": "1",
            "total_tax": "4",
            "order_tax": "3",
            "shipping_tax": "1",
            "orders": "1"
        },
        {
            "datetime_anchor": "2018-12-01 00:00:00",
            "time_interval": "2018-48",
            "tax_codes": "1",
            "total_tax": "18.37",
            "order_tax": "13.37",
            "shipping_tax": "5",
            "orders": "1"
        },
        {
            "datetime_anchor": "2018-11-11 00:00:00",
            "time_interval": "2018-45",
            "tax_codes": "1",
            "total_tax": "24",
            "order_tax": "24",
            "shipping_tax": "0",
            "orders": "1"
        }
    ]
}
```

Reply:
My hunch on this is that we shouldn't limit to 2 decimal places and let `formatCurrency` handle this on the client side since some currencies don't use decimals and others use up to 3 decimal places.

This is the way it's handled in the other endpoints as of right now, so I'll go ahead and get this one merged in, but I'll check with @peterfabian to make sure this is the best way to handle this and we can update across the board if needed.